src/h2load.cc

```diff
@@ -1199,6 +1319,12 @@ Worker::~Worker() {
   ev_loop_destroy(loop);
 }
 
+void Worker::stop_all_clients() {
+  for(auto client: clients) {
+    client->~Client();
```

Comment:
I see, looking at the code it seems there is quite a bit of non-RAII behavior. I think you want to do something like what is done in https://github.com/nghttp2/nghttp2/blob/master/src/h2load.cc#L746 rather than explicitly run the destructor though, so you can reuse the same teardown path as the existing code.